lib/matplotlib/textpath.py

```diff
@@ -243,25 +242,30 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
 
         # Gather font information and do some setup for combining
         # characters into strings.
-        for x1, y1, dvifont, glyph, width in page.text:
-            font, enc = self._get_ps_font_and_encoding(dvifont.texname)
-            char_id = self._get_char_id(font, glyph)
-
+        for text in page.text:
+            font = get_font(text.font_path)
+            char_id = self._get_char_id(font, text.glyph)
             if char_id not in glyph_map:
+                glyph_name_or_index = text.glyph_name_or_index
```

Comment:
Might be moved down to l.252 and I don't think you stll need the comment there.

Reply:
done